README.md

```diff
@@ -71,6 +71,17 @@ The Jaeger Operator *might* work on versions outside of the given range, but whe
 | v1.23           | v1.19+               |
 | v1.22           | v1.18 to v1.20       |
 
+### Jaeger Operator vs. Strimzi Operator
```

Comment:
It might be better to use the table above, so that we update only one place when we need to perform a release.

Reply:
Done!